src/_scss/pages/homepage/features/_featurePane.scss

```diff
@@ -0,0 +1,116 @@
+.feature-pane {
+    .feature-pane__wrapper {
+        display: block; 
+        margin: rem(60) 0 rem(100);
+        width: 100%;
+
+        @include media($medium-screen) {
+            margin: rem(120) 0 rem(60);
+        }
+    }
+
+    .feature-pane__title {
+        width: 100%; 
+        text-align: center; 
+        border: 1px solid rgba(214,215,217,0.4);
+        line-height: rem(0.1);
+        font-size: rem(16);
+        margin: 10px 0  20px;
+        color: #717171;
+
+        & span {
+            background:#fff; 
+            padding:0 10px; 
+        }
+    }
+
+    .feature-pane__content-wrapper {
+        display: flex;
+        justify-content: space-around;
+        margin-bottom: rem(20);
+        flex-wrap: wrap;
+    }
+
+    .feature-pane__content {
+        flex-basis: 100%;
```

Comment:
Rather than use `flex-basis`, please use the `flex` [shorthand property](https://developer.mozilla.org/en-US/docs/Web/CSS/flex) instead. This makes grow and shrink factors explicit (IE11 may not use the defaults you're expecting with other browsers).